public/css/content.css

```diff
@@ -0,0 +1,66 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
+
+:root {
+  --color-wildhero-background: #f9f9f9;
+  --color-wildhero-compose-button: #afeab1;
+  --color-wildhero-selected-tab-left: #c9f6c9;
+  --color-wildhero-search-bar: #eeeeee;
+  --color-wildhero-selected-tab-top: #3e8903;
+  --color-wildhero-read-email: #f7f7f7;
+  --color-wildhero-unread-email: #ffffff;
+  --color-wildhero-selected-email: #d3f0d3;
+}
+
+/* hide */
+header div[role='button'][jsaction] {
+  display: none;
+}
+
+/* overrides theme */
+div[role='button'].T-I.T-I-KE.L3 {
```

Comment:
**Suggestion:**
Wherever needed consider using Styled Components for consistent styling and easier maintenance by co-locating styles with components.